.kokoro/build.sh

```diff
@@ -156,7 +156,7 @@ case ${JOB_TYPE} in
           -Penable-integration-tests \
           -Pnative \
           -fae \
-          verify
+          test
```

Comment:
Would this skip ITs now?

Reply:
GraalVM checks do not run the failsafe (IT) plugin since the native plugin is compatible to run with it. We set the surefire plugin to use the IT prefix (https://github.com/googleapis/java-shared-config/blob/8d7f75de442cb3a274004faae430e9edf60730f3/pom.xml#L806). 

Example in the split-repos (Java-Vision): https://github.com/googleapis/java-vision/blob/359492111a9b5d20de04e4e98ecb6374e63cfb9d/.kokoro/build.sh#L74